### ISSUE TITLE
overlines get positioned wrong when using scaling, especially \huge

### ISSUE THREAD

**Issue by Mamadsho** [mathjax/MathJax#2983]
### Over elements get positioned too high when using scaling
**`\normalsize`** VS **`\huge`**

![изображение](https://user-images.githubusercontent.com/30808439/210350201-fa0c5a85-c1ee-4245-9e63-8c3dd58eb0dc.png)

### Steps to Reproduce:

1. In MathJax Live Demo using `\huge` with `\vec`, for example like this:
`$$ \huge y = \vec x  -  \vec x ^2 $$`

Interestingly, if one uses sub- or superscript the position gets fixed, as in the `\vec x ^2`

### Technical details:

* Client OS: Windows 11
* Application: Obsidian, WebDemo in Firefox 108.0.1

### Supporting information:

 * _It seems to be working on some other versions_

**Comment by dpvc** [comment]
Yes, this is a bug in v3.2.2 (not in 3.2.0 or earlier).  It is resolved in the 4.0.0-alpha release, and will be in the official 4.0 release.

**Comment by dpvc** [comment]
I was mistaken, it was not fully resolved in v4.0.0-alpha.  I have made a PR that fixes the issue.

**Comment by dpvc** [comment]
Fixed in v4.0
